scope.cpp

```diff
@@ -495,7 +495,7 @@ bool Scope::CalibrationFlipRequiresDecFlip()
     return m_calibrationFlipRequiresDecFlip;
 }
 
-bool Scope::HasHPDecEncoder()
+bool Scope::HasHPEncoders()
 {
     return pConfig->Profile.GetBoolean("/scope/HiResEncoders", false);

```

Comment:
minor: ideally it would be best to have a member variable containing this attribute -- we can use pConfig as our backing store for settings, but for runtime checks use member variables.  The reason is that going though pConfig to get s simple boolean attribute is a heavyweight mechanism ("[premature pessimization](https://elegantchaos.com/2014/10/27/premature-pessimization-and-the-like.html)") In the cases where we use this (GA) we don't really care, so this is probably ok to leave as-is. The motivation for changing it would be to (a) not perpetuate the pattern of using pConfig in place of member variables for object state, and (b) have an efficient function that could be used if this ever needed to be called on the guiding path (outside GA).